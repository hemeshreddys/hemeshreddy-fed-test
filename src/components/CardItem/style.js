import Styled from 'styled-components'

export const Card = Styled.div `
    display: flex;
    flex-direction: row;
    align-items: center;
    width: 100%;
    margin-bottom:5px;
    padding: 10px 20px;
    background-color:#f44336;
    border-radius: 5px;
    color: #ffffff;    
    @media (min-width: 1024px) {
        width: 30%;
    }
    img{
        border-radius: 75px;
        max-width: 25%;
    }
    .users{
        margin-left:20px;
        h2, p { 
            margin: 0;
        }
    }
`
