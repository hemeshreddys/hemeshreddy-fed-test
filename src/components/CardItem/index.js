import React from "react";
import * as style from "./style.js";

const CardItem = (props) => {
    return (
        <style.Card>
            <img src={props.imgurl} />
            <div className="users">
            <h2>{props.name}</h2>
            <p>{props.email}</p>
            <p>{props.location}</p>
            </div>
        </style.Card>
    )
}

export default CardItem;

