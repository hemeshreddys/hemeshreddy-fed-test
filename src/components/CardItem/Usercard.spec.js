import React from 'react';
import CardItem from './index.js';

describe('CardItem', () => {
	it('renders the CardItem', () => {
		const actual = shallow(<CardItem />);
		expect(actual.get(0)).toMatchSnapshot();
		
	});
});
