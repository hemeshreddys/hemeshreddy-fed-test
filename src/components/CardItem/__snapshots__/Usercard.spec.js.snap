// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardItem renders the CardItem 1`] = `
<styled.div>
  <img />
  <div
    className="users"
  >
    <h2 />
    <p />
    <p />
  </div>
</styled.div>
`;
